app/src/org/commcare/android/util/FileUtil.java

```diff
@@ -395,4 +396,20 @@ public static String getDumpDirectory(Context c){
                 return null;
             }
         }
+
+    public static Properties loadProperties(Context c) throws IOException {

```

Comment:
Hey,

out of curiosity, this will still result in our submission credentials being pretty cleanly available in the dissasembled APK, right? What is their best practice around protecting your credentials?